datarobot_batch_scoring/network.py

```diff
@@ -372,6 +373,16 @@ def run(self, dry_run=False):
                 self.ui.info('{} responses sent | time elapsed {}s'
                              .format(i, time() - t0))
 
+                if time() - last_report > 10:
```

Comment:
This is nice for the windows people. Could we do it more often? Maybe every 5 seconds?